EIPS/eip-7800.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7800
+title: eth/70 - Sharded Blocks Protocol
+description: Replaces block range with a bitlist representing 1-million-block spans in the handshake, with probabilistic shard retention
+author: Ahmad Bitar (@smartprogrammer93) <smartprogrammer@windowslive.com>, Giulio Rebuffo (@Giulio2002)
+discussions-to: https://ethereum-magicians.org/t/eip-7800-eth-70-sharded-blocks-protocol/21507
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7801-eth-70-sharded-blocks-protocol/21507
```
Added assigned EIP number and changed category in Eth Magicians.  Please use the discussions template: https://ethereum-magicians.org/t/template-for-discussion-to-threads/20347

Reply:
mh - is the discussion template only a suggestion? I saw many discussion not doing it. anyway it seems to me a bit overkill to do it